pandas/core/groupby.py

```diff
@@ -2454,7 +2562,10 @@ def transform(self, func, *args, **kwargs):
 
         # if string function
         if isinstance(func, compat.string_types):
-            return self._transform_fast(lambda : getattr(self, func)(*args, **kwargs))
+            if func in _cython_transforms:
```

Comment:
``_intercept_cython`` should handle these types of things no? and if not, then the syntax should be natural and transparent and not lots of if/thens